contrib/cirrus/runner.sh

```diff
@@ -375,6 +375,14 @@ dotest() {
         podman)  localremote="local" ;;
     esac
 
+    # We've had some oopsies where tests invoke 'podman' instead of
+    # /path/to/built/podman. Let's catch those.
+    rm -f /usr/bin/podman /usr/bin/podman-remote
+    fallback_podman=$(type -p podman || true)
+    if [[ -n "$fallback_podman" ]]; then
+        warn "Found fallback podman '$fallback_podman' (this isn't an error, it just means we can't catch bugs in which tests invoke 'podman' instead of /path/to/src/built/podman)"
```

Comment:
> The warning will likely never be seen by
a human, but I may check for them periodically.

Not sure why this would not be a hard error?

Reply:
That was my first draft but I chickened out, because I don't know enough about the setup of the CI VMs. Since I have to re-push anyway, I might go that route.